### PR TITLE
BaSP-TG-2: Super Admin - Add/Edit Form

### PR DESCRIPTION
**Pull request by elias-92**
Points to test:
-Add route on Layout Component
-Create add/edit form including: First Name, Last Name, Email, Password
-On edit functionality should show previous info
-Warning/error messages for edit and add

### CODE REVIEW

File: src/Components/SuperAdmins/Form/index.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+        console.log(error);
+      }
+    }
+  }, []);
+
+  const editSuperAdmin = async (userId) => {
+    try {
+      await fetch(`${process.env.REACT_APP_API_URL}/super-admins/${userId}`, {
+        method: 'PUT',
+        headers: {
```
Comment (LucianaPrat) [inline src/Components/SuperAdmins/Form/index.js:48]: headers are not necessary

File: src/Components/SuperAdmins/index.js [hunk 2]
```diff
@@ -1,11 +1,59 @@
...
+
+  const deleteSuperAdmin = async () => {
+    try {
+      await fetch(`${process.env.REACT_APP_API_URL}/super-admins/${id}`, {
+        method: 'DELETE'
+      });
+      setSuperAdmins([...superAdmins.filter((listItem) => listItem._id !== id)]);
+      setModal(false);
+    } catch (error) {
+      console.log(error);
```
Comment (LucianaPrat) [inline src/Components/SuperAdmins/index.js:31]: console log is not necessary

File: src/Components/SuperAdmins/Form/index.js [hunk 3]
```diff
@@ -0,0 +1,184 @@
+import { useEffect, useState } from 'react';
+import styles from './form.module.css';
+import ModalAlert from '../ModalAlert';
+
+const FormSuperAdmins = () => {
+  const paramsURL = new URLSearchParams(window.location.search);
```
Comment (martumoser) [inline src/Components/SuperAdmins/Form/index.js:6]: You have to use react props to get the id of the superadmin selected. The father component should sent the id of the superadmin selected as a prop.

File: src/Components/SuperAdmins/Form/index.js [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+          name: data.data.name,
+          lastName: data.data.lastName,
+          email: data.data.email,
+          password: data.data.password
+        });
+      } catch (error) {
+        console.log(error);
+      }
+    }
+  }, []);
```
Comment (martumoser) [inline src/Components/SuperAdmins/Form/index.js:42]: Why you are doing this? this info about the super admin should be received as a prop

File: src/Components/SuperAdmins/Modal/index.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
+import styles from './modal.module.css';
+
+const ModalDelete = ({ modal, setModal, deleteSuperAdmin }) => {
```
Comment (martumoser) [inline src/Components/SuperAdmins/Modal/index.js:3]: Why you are including this on this pr? delete functionality is on another PR

**Review by martumoser — CHANGES_REQUESTED** [review]

**Review by martumoser — APPROVED** [review]

**Review by LucianaPrat — APPROVED** [review]

**Comment by NicolasGulam** [comment]
Done a simple comment in trello

**Review by NicolasGulam — APPROVED** [review]

**Pull request merged**